### PR TITLE
DOCSP-43721-migration-verifier-disclaimer

### PR DESCRIPTION
**Pull request by amalhotra-mdb**
## DESCRIPTION

Migration Verifier has the following limitation, according to the [Migration Verifier GitHub](https://github.com/mongodb-labs/migration-verifier#limitations): 

> The verifier's iterative process can handle data changes while it is running, until you hit the writesOff endpoint. However, it cannot handle DDL commands. If the verifier receives a DDL change stream event (drop, dropDatabase, rename), the verification will fail. If an untracked DDL event (create, createIndexes, dropIndexes, modify) occurs, the verifier may miss the change.

On the page instructing users on how to [Verify using Migration Verifier](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/verification/verifier/#verify-with-migration-verifier) and the [Verify Data Transfer](https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/verification/#std-label-c2c-verify-with-verifier) page, there is no disclaimer that Migration Verifier cannot handle DDL commands. This PR adds this disclaimer to both pages and updates the already existing link to the Migration Verifier GitHub to note the Limitation explanations in the GitHub readme. 

## STAGING

- https://deploy-preview-445--docs-cluster-to-cluster-sync.netlify.app/reference/verification/#migration-verifier

- https://deploy-preview-445--docs-cluster-to-cluster-sync.netlify.app/reference/verification/verifier/#verify-with-migration-verifier

## JIRA
https://jira.mongodb.org/browse/DOCSP-43721

## SELF-REVIEW CHECKLIST

- [ ] Does each file have 3-5 taxonomy facet tags?
  See the [taxonomy tagging instructions](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) and this [example PR](https://github.com/10gen/cloud-docs/pull/5042/files) 
- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## EXTERNAL REVIEW REQUIREMENTS

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/reference/verification.txt [hunk 1]
```diff
@@ -99,11 +99,14 @@ Migration Verifier connects to the source and destination clusters and performs
...
+
 .. important::
 
    Migration Verifier is an experimental and unsupported tool.
 
-   For installation instructions, see
+   For installation instructions and usage limitations, see
    `GitHub <https://github.com/mongodb-labs/migration-verifier>`__.
 
 Unlike other verification methods, Migration Verifier can run concurrent
```
Comment (jmd-mongo) [inline source/reference/verification.txt:112]: [typo]

I know this isn't part of the original ask, but could we fix this typo here? This should be "...can run concurrently...".

File: source/reference/verification.txt [hunk 2]
```diff
@@ -99,11 +99,14 @@ Migration Verifier connects to the source and destination clusters and performs
 a series of verification checks, comparing documents, views, and indexes to
 confirm the sync was successful.
 
+Migration Verifier does not support DDL operations. When using the Migration
+Verifier, ensure that your source cluster does not use DDL commands.
```
Comment (jmd-mongo) [inline source/reference/verification.txt:103]: [question]

Is there a way to get more specific about what using DDL commands means in this context? 

Are we saying users shouldn't run DDL operations on the source cluster during migration? Are there additional considerations?

**Review by jmd-mongo — COMMENTED** [review]
Thanks for this helpful addition, @amalhotra-mdb! I've left a question for your consideration and noted a small typo I think we can address.

Thanks!
Joe

File: source/reference/verification.txt (hunk 2 above)
Comment (amalhotra-mdb) [reply source/reference/verification.txt]: Thanks for taking a look at this and for raising this questions, @jmd-mongo ! I tried to reword this sentence now to clarify that no DDL commands can be running on the source cluster when the user is verifying their data using Migration Verifier – lmk if you have any changes or suggestions!

I don't believe there are any additional considerations – the Limitations section in the [Migration Verifier GitHub ReadMe](DOCSP-43721-migration-verifier-disclaimer) I _think_ gives the user all the specifics they'd need for this consideration.

**Review by jmd-mongo — APPROVED** [review]
Thanks for this update. This one LGTM at this point.

Thanks!
Joe

**Pull request merged**